ARTICLE-2.markdown

```diff
@@ -12,7 +12,7 @@ A Membership shall immediately terminate upon the death of a Member.
 ### Section 2.3  Expulsion
 
 (a)	A Member may for any lawful reason be expelled from the
-        Corporation by unanimous consensus of the Member Body.
+        Cooperative by unanimous consensus of the Member Body.
```

Comment:
David, when number of members increase, this is going to be an issue. Collecting all members and taking consensus will be a herculean task